index.php

```diff
@@ -44,23 +33,28 @@
 </style>
 
 <form action="Results1.php" method="post" name="Form1">
+<!-- title of page -->
 <center>
 	<font size="18" color = "red" face = "Tw Cen MT">UMBC Advising Kiosk</font>
 </center>
+
+<!-- places a box around the login elements -->
 <div class = "loginBox">
 <h1>
+	<!-- input fields fName, lName, StudentID -->
 	<font size="5" face="Serif">
 	<br>
-	FIRST NAME: <input type="text" size="12" maxlength="20" name="tfNameF" required><br>
+	FIRST NAME: <input type="text" size="12" maxlength="20" name="tfNameF" placeholder="John" required><br>
```

Comment:
I would modify the placeholder here, make it obvious that this is an example.

```html
<input type="text" size="12" maxlength="20" name="tfNameF" placeholder="e.g. John" required><br>
```

Same goes for the other two text fields.